contrib/ray/raycluster_example.yaml

```diff
@@ -1,49 +1,29 @@
 apiVersion: ray.io/v1
 kind: RayCluster
 metadata:
-  name: kubeflow-raycluster
+  name: raycluster-istio
```

Comment:
Renaming this breaks the test. can you keep it as kubeflow-raycluster ?

Reply:
And we should install to the default user namespace if that is not already the case.